.azure-pipelines-templates/matrix.yml

```diff
@@ -61,17 +61,6 @@ jobs:
       artifact_name: "SGX_Perf"
       ctest_filter: "${{ parameters.test.perf.ctest_args }}"
 
-  # Fuzzing
```

Comment:
This looks like its just removing the Fuzzing test entirely - where is it added to the Daily?

Reply:
Sorry, git mistake, thanks for catching, will fix.